tools/editor/plugins/script_editor_plugin.cpp

```diff
@@ -827,6 +827,24 @@ void ScriptEditor::_close_current_tab() {
 
 }
 
+void ScriptEditor::_close_docs_tab() {
+
+	int child_count = tab_container->get_child_count();
+	for (int i = 0; i<child_count; i++) {
+
+		EditorHelp *ste = tab_container->get_child(i)->cast_to<EditorHelp>();
+
+		if (ste) {
+			_close_tab(i);
+			i--;
```

Comment:
Wouldn't it be cleaner to process the tabs from the last one to the first one, and just close the relevant one, instead of manipulation `i` and `child_count` from within a `for` loop?